models/multiphase/d2q9_pf_velocity/Dynamics.R

```diff
@@ -141,8 +144,10 @@ AddGlobal(name="InletFlux", comment='pressure loss', unit="1m2/s")
 AddGlobal(name="TotalDensity", comment='Mass conservation check', unit="1kg/m3")
 AddNodeType(name="SpikeTrack", group="ADDITIONALS")
 AddNodeType(name="BubbleTrack", group="ADDITIONALS")
+AddNodeType(name="WAVE", group="ADDITIONALS")
```

Comment:
Since other node types are not all caps, I would expect this one to be just "Wave" for consistentcy